python/setup_project/storage_locations_page.py

```diff
@@ -1,301 +0,0 @@
-# Copyright (c) 2013 Shotgun Software Inc.
```

Comment:
The old storage locations page could show up multiple times, once for each required root that didn't exist (or was invalid) in SG. The UI was constructed programmatically. This has been replaced with a single page that shows all the roots to map. It is editable via designer now.